app/src/main/AndroidManifest.xml

```diff
@@ -40,15 +40,15 @@
             </intent-filter>
         </activity>
         <activity
-            android:name=".TUC_Info"
+            android:name=".TucInfo"
             android:label="TUCInfo">
         </activity>
         <activity
-            android:name=".TUC_Photos"
+            android:name=".TucPhotos"
             android:label="Photos">
         </activity>
         <activity
-            android:name=".CECH_Info"
+            android:name=".CechInfo"
```

Comment:
The new UpperCamelCase naming convention is inline with Kotlin naming conventions.